### DISCUSSION TITLE
Gamin Vario with OSMand?

### DISCUSSION THREAD

**Discussion by SportTester** [osmandapp/OsmAnd#12348] [Ideas]
Could you add the ability to add the Garmin Vario device radar to the app please? Ride Gps has it! 

This devices keeps you safe on the road and more developer need to add it to their app. :) It can track cars, bikes, etc 140m away from you and people are crazy about it. 

The Varia is very good at identifying a relevant approach. If your bike is stationary and you walk up to it from behind it'll count that as an approach. And, as we've said, other vehicles - motorbikes, cars, vans and beyond - and cyclists are detected with ease. 

Here is a picture of what it looks like and be so good if OSM paid and free has this. 
[https://ridewithgps.com/help/wp-content/uploads/2020/05/iphone_orange_lane.jpg](url)

**Comment by tmsbr** [comment]
I would ask for ANT+ Radar support, not only for Varia, but for Magene L508, Bryton Gardia R300L, IGPSPORT SR30 Radar's too. 

Each one of them seems to have a different BLE protocol linked to the their own apps, but all of them use the same ANT+ protocol.

Currently no cycling app offer support for all of them in ANT+.

For cyclists it's a must have support for Radar's inside the app, this devices are very popular now and increase safety.
